### ISSUE TITLE
Add example for models.py

### ISSUE THREAD

**Issue by dkdndes** [UniversitaDellaCalabria/django-form-builder#8]
The documentation for https://django-form-builder.readthedocs.io/en/latest/usage/dynamic-forms/configuration.html does not explain in full how models.py can be used. It would help to see an example, or add the model fields to the model in the documentation.

**Comment by DarnelleF** [comment]
I must agree with @dkdndes, documentation is a little limiting. and would love to implement into a project but might as well build myself.

**Comment by francesco-filicetti** [comment]
you are right. I will provide as soon as possible

**Comment by dkdndes** [comment]
If you get a pull request in 2025, would you be able to review it in 2025?
